README.md

```diff
@@ -0,0 +1,186 @@
+<p align="center">
+  <img src="./docs/AIMY_banner.png"width="100%" alt="AIMY Logo" align="center"/>
+</p>
+
+# Low-level Ball Launcher Control 
+Ball Launcher Beepy is a low-level control environment for remote table tennis ball launching. It features 
+- hardware server functions for controlling of motors and servomotors via Python
+- client APIs for remote launching via Python and C++ 
+
+## Installation
+Installation under Ubuntu 16.04 or Ubuntu 18.04
+
+### Requirements
+* libzmq (ZeroMQ)
+* protobuf (Protocol buffers)
+
+The server side additionally needs
+* xinabox-OC03
+* xinabox-OC05
+
+```bash
+sudo apt install libzmq3-dev
+sudo apt install protobuf-compiler libprotobuf-dev
+
+# for server side only (TODO: Does this need a sudo?)
```

Comment:
it depends if it is installed for the current user, the current virtual env, or globally (but we can assume this to be common knowledge, so no need for this TODO)